index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lesson Learned - A.AFL</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header id="cabecalho">
+      <h1 id="titulo">Lesson Learned</h1>
+      <h2>Alan Albuquerque Ferreira Lopes</h2>
+      <nav>
+        <a href="#licoes aprendidas">Lições</a>
+        <a href="#rodape">Contato</a>
+        <a href="https://offpepe.github.io/">Portfolio</a>
+      </nav>
+    </header>
+    <section class="photo_section">
+      <img
+        src="src/imgs/me_photo.jpeg"
+        alt="Foto de Alan Albuquerque"
+        id="minha_foto"
+      />
+    </section>
+    <div id="texts">
+    <main>
+      <article class="lado-esquerdo">
```

Comment:
Alan, essa sugestão é apenas caso vc ache necessário colocar os dois `h3` ("Meu Aprendizado" e "O porquê continuo avançando") na mesma altura. A sugestão aqui seria criar um `div` que englobe tanto o `article` quanto o `aside` e colocar o `display` de `div` como `flex.` Faz sentido pra vc?
```suggestion 
      <div id = "article-and-aside"
      <article class="lado-esquerdo">
```
![Screenshot from 2021-02-27 22-39-21](https://user-images.githubusercontent.com/78616220/109405199-b371ff00-794c-11eb-9dd3-79db411806db.png)